### PR TITLE
[1.x] Add QRCode to API Tokens

### PR DESCRIPTION
**Pull request by luizeof**
This PR adds a QR Code on the New API Token screen for mobile users scan the Token.

jetstream-api-svg

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
         "ext-json": "*",
         "illuminate/support": "^8.0",
         "jenssegers/agent": "^2.6",
-        "laravel/fortify": "^1.0"
+        "laravel/fortify": "^1.0",
+        "bacon/bacon-qr-code": "^2.0"
```
Comment (driesvints) [inline composer.json:22]: Alphabetically order this dependency.

Comment (luizeof) [reply composer.json]: Done @driesvints

File: composer.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
     ],
     "require": {
         "php": "^7.3",
+        "bacon/bacon-qr-code": "^2.0",
         "ext-json": "*",
```
Comment (bonzai) [inline composer.json:19]:
```suggestion
        "ext-json": "*",
        "bacon/bacon-qr-code": "^2.0",
```

**Comment by taylorotwell** [comment]
Thanks for your pull request to Laravel!

Unfortunately, I'm going to delay merging this code for now. To preserve our ability to adequately maintain the framework, we need to be very careful regarding the amount of code we include. 

If possible, please consider releasing your code as a package so that the community can still take advantage of your contributions! 

If you feel absolutely certain that this code corrects a bug in the framework, please "**@**" mention me in a follow-up comment with further explanation so that GitHub will send me a notification of your response.

**Pull request closed without merging**